### PR TITLE
update Friday schedule

### PR DESCRIPTION
**Pull request by JessicaS11**

### CODE REVIEW

**Comment by jomey** [comment]
We can also shorten the follow up CryoHub slot before the survey.
Don't see a need for 30 mins there and 15 mins works fine

File: book/schedule.yaml [hunk 1]
```diff
@@ -162,17 +162,17 @@ tabs:
...
+    time: 10:00 - 10:30
   - title: Sharing Project Outcomes - Part I
     description: Sharing what you and your teams accomplished during our time together.
-    time: 10:00 - 11:30
-    leads:
-    - Anthony Arendt
-  - title: SURVEY FEEDBACK
-    description: Complete our online survey so that we can improve our program.
-    time: 11:30 - 12:00
+    time: 10:30 - 11:30
```
Comment (aaarendt) [inline book/schedule.yaml]: Should be 10:30 - 12:00

**Comment by jomey** [comment]
> We can also shorten the follow up CryoHub slot before the survey. Don't see a need for 30 mins there and 15 mins works fine

Also make @tsnow03 the main presenter of this session

**Comment by JessicaS11** [comment]
> Don't see a need for 30 mins there and 15 mins works fine

I suspect we'll start a little late, so figure it will work out. @aaarendt thoughts?

**Review by aaarendt — APPROVED** [review]

**Pull request merged**